rosdep/python.yaml

```diff
@@ -6095,6 +6095,9 @@ python3-pkg-resources:
   openembedded: [python3-setuptools@openembedded-core]
   rhel: ['python%{python3_pkgversion}-setuptools']
   ubuntu: [python3-pkg-resources]
+python3-progressbar:
+  ubuntu:
+    focal: [python3-progressbar]
```

Comment:
This package appears to be available from xenial and newer. We shouldn't specifically target focal. 

This also doesn't cover all our main platforms as outlined in the contributing guide: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules